recipe/meta.yaml

```diff
@@ -21,24 +21,28 @@ source:
     - no_fileutils_test.patch
 
 build:
-  number: 2
-  skip: True  # [win or not py36]
+  number: 3
+  skip: false  # [win or not py36]
```

Comment:
This needs to be skip: true # [not py36] and a rerender

Reply:
What about taking skip out altogether since python2.5 also passed?

Reply:
We need the skip because we only want to build with Python 3.6